backend/app/adapter/sqldb/user_short_link_integration_test.go

```diff
@@ -57,18 +58,23 @@ func TestListShortLinkSql_FindAliasesByUser(t *testing.T) {
 		{
 			name: "aliases found",
 			userTableRows: []userTableRow{
-				{email: "test@example.com"},
+				{
+					id:    "test",
+					name:  "test user",
```

Comment:
Shouldn't the name be the same?
```suggestion
					name:  "mockedUser",
```

Reply:
I agree

Reply:
Will make the changes